### PR TITLE
Update thethingsnetwork.markdown

### PR DESCRIPTION
**Pull request by martin3000**
Suggest MQTT for TTNv3

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/thethingsnetwork.markdown [hunk 1]
```diff
@@ -16,6 +16,7 @@ ha_integration_type: integration
 
 <div class='note warning'>
   This integration only supports TTNv2 that has been definitively switched off in 2022 (deprecated).
+  For TTNv3 use the MQTT-Integration.
```
Comment (FletcherAU) [inline source/_integrations/thethingsnetwork.markdown]: Should this link to `/integrations/mqtt`?

Comment (martin3000) [reply source/_integrations/thethingsnetwork.markdown]: Yes it can be linked.

Comment (c0ffeeca7) [inline source/_integrations/thethingsnetwork.markdown]:
```suggestion
  For TTNv3, use the [MQTT integration](/integrations/mqtt).
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @martin3000 :+1:

**Pull request merged**